includes/class-wc-payments-redirect-service.php

```diff
@@ -122,9 +122,10 @@ public function redirect_to_account_link( array $args ): void {
 	 * Note that this function immediately ends the execution.
 	 *
 	 * @param string|null $error_message Optional error message to show in a notice.
-	 * @param string      $from          Optional source of the redirect.
+	 * @param string|null $from Optional source of the redirect.
+	 *                     Will fall back to keeping the `from` parameter in the current request URL, if present.
 	 */
-	public function redirect_to_connect_page( ?string $error_message = null, string $from = '' ): void {
+	public function redirect_to_connect_page( ?string $error_message = null, string $from = null ): void {
```

Comment:
```suggestion
	public function redirect_to_connect_page( ?string $error_message = null, ?string $from = null ): void {
```

Reply:
We should change the type to nullable if we passing `null` as default value

Reply:
Done in https://github.com/Automattic/woocommerce-payments/pull/9036/commits/b2f1af4b745ab68b2bbd9e42537c8531b2667bda